infra/resources/_modules/apim/apim.tf

```diff
@@ -0,0 +1,37 @@
+module "apim_v2_wallet_api" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v8.12.2"
+
+  name                  = format("%s-ioweb-wallet", var.project_legacy)
+  api_management_name   = var.apim.name
+  resource_group_name   = var.apim.resource_group_name
+  product_ids           = [var.product_id]
+  subscription_required = false
```

Comment:
Should these APIs be accessible from external world?

Reply:
no just from IO web